config.sample.yml

```diff
@@ -50,7 +50,7 @@ listeners:
 #    # (Optional) Settings for allowing users to sign in via OAuth.
 #    client_id: foo
 #    client_secret: bar
-#    redirect_uri: https://example.com/bridge_oauth/
+#    redirect_uri: https://example.com/oauth/
```

Comment:
You'll need to update it in https://github.com/matrix-org/matrix-hookshot/blob/main/src/config/Defaults.ts#L69 too, as this file is generated from that.